src/v/pandaproxy/handlers.cc

```diff
@@ -206,6 +210,31 @@ create_consumer(server::request_t rq, server::reply_t rp) {
       });
 }
 
+ss::future<server::reply_t>
+remove_consumer(server::request_t rq, server::reply_t rp) {
+    auto group_id = kafka::group_id(rq.req->param["group_name"]);
+    auto member_id = kafka::member_id(rq.req->param["instance"]);
+
+    return rq.ctx.client.remove_consumer(group_id, member_id)
```

Comment:
looks like we could just coawait and remove the next 2 lines since you are already doing a try-catch.